apps/webapp-source/Gopkg.toml

```diff
@@ -0,0 +1,7 @@
+[prune]
```

Comment:
Unnecessary. This project has no external dependencies and the `dep` materials should be removed; it's confusing.